documentation/running_bolt_commands.md

```diff
@@ -27,6 +27,26 @@ Invoke-BoltCommand -Command 'Get-Location' -Targets servers
 > 🔩 **Tip:** If a command contains spaces or special shell characters, wrap
 > the command in single quotation marks.
 
+### Run a quoted command
+
+If you need to run a command with quotes inside, there are a few considerations to keep in mind. You will need properly escape the quotes according to the whether you are running in `bash` or in `powershell`.
+
+In a `bash` shell, you'll need to use backslashes `\` or double the quote you're using in order for the quotes to pass through.
+
+_\*nix shell command_
+
+```shell
+bolt command run "Get-WMIObject Win32_Service -Filter ""Name like '%mon'""" -t localhost
+```
+
+In a `powershell` shell, you'll need to use a combination of escape characters and doubling quotes in order for the quotes to pass through. In the example below, we use two double quotes to quote the value being passed to `Filter`, however we need to pass a backslash in order to have the ruby code below it accept the command.
```

Comment:
```suggestion
In a PowerShell shell, use a combination of backslashes `\` and doubling of quotation marks. The example below uses two double quotation marks to quote the value being passed to `Filter`, however the example also uses a backslash so that the Ruby code below the value accepts the command.
```

Reply:
I'm not 100% on "The Ruby code below it". Are we talking about the underlying code that makes up Bolt here?

Reply:
The ruby argument parser will properly accept the quotes if they are escaped. This is an artifact of passing tokens through two levels of programs. 

Maybe: `so that the Ruby argument parser accepts the command`

Reply:
Okay, I made a small edit. How does that look?